src/components/Button.js

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ name }) => (
-  <div id="button">
-    { name }
+const Button = ({ name, wide, color }) => (
+  <div
+    className="button"
+    style={{
+      backgroundColor: color ? '#e0e0e0' : '',
+      width: wide ? '50%' : '',
+    }}
+  >
+    {name}
   </div>
 );
 
 Button.propTypes = {
   name: PropTypes.string.isRequired,
+  color: PropTypes.bool,
+  wide: PropTypes.bool,
+};
+
+Button.defaultProps = {
+  color: false,
+  wide: false,
 };
```

Comment:
- Couple of things to note here in this component, first thing, we need to make sure that our color prop takes the string orange. As you can see for the prop wide, is explicitly called for it to be a boolean whilst the color one not. So let's work the display logic around the fact that color's prop value is 'orange' instead of a boolean.

Reply:
Thanks, @jofer86. The change has been made as requested.